pylzy/lzy/api/v1/local/runtime.py

```diff
@@ -126,7 +126,7 @@ async def __exec_call(self, call: LzyCall):
                 args_paths=arg_descriptions,
                 kwargs_paths=kwarg_descriptions,
                 output_paths=ret_descriptions,
-                exception_path=e_description,
+                exception_path=exc_description,
```

Comment:
Well, why are you putting "description" into "path"?

Reply:
@NataliaPonomarevaMM , up